BlockBehavior/BehaviorHarvestable.cs

```diff
@@ -122,7 +122,8 @@ public override void OnBlockInteractStop(float secondsUsed, IWorldAccessor world
 
                 if (harvestedBlock != null)
                 {
-                    world.BlockAccessor.SetBlock(harvestedBlock.BlockId, blockSel.Position);
+                    if (!properties["exchangeBlock"].AsBool(false)) world.BlockAccessor.SetBlock(harvestedBlock.BlockId, blockSel.Position);
+                    else world.BlockAccessor.ExchangeBlock(harvestedBlock.BlockId, blockSel.Position);

```

Comment:
This makes it so you can exchange blocks when harvested so we don’t have to reset the berry bush entities when the block is harvested.